### ISSUE TITLE
Automated test for validating we have all localized files

### ISSUE THREAD

**Issue by crutkas** [Microsoft/WindowsTemplateStudio#645]
We need a way to validate we have all the localized files

- all json files should deserialize
- localized json files should not contain default language content
- localized md files should not contain default language content
- all equivalent resw files should have same key entries
- localized resw file should not contain default language content

**Comment by ralarcon** [comment]
Can you drill down a bit in what do you have in mind wiht "not be default"?, is this to make a process wich changes the Culture and test that all files loads / deserialized properly?

**Comment by mrlacey** [comment]
@ralarcon I've updated the bullet points in the OP based on a conversation I had with @crutkas yesterday to add clarity.
There are two things here:
- Making sure we have the correct files existing and with valid JSON
- While we can't check that things have been translated correctly we can check that they've not been left with the default language. i.e. Know that they're definitely wrong.


This needs to tie-in with #646 too. Identifying what is new and hasn't ever been translated and what has been changed and needs the translations updated shouldn't be different tasks.
